src/Process/LayeredMaterialNodeProcessing.js

```diff
@@ -228,11 +228,19 @@ export function updateLayeredMaterialNodeImagery(context, layer, node) {
     }
 
     const currentLevel = node.material.getColorLayerLevelById(layer.id);
-    const zoom = node.getCoordsForLayer(layer)[0].zoom || node.level;
-    const targetLevel = chooseNextLevelToFetch(layer.updateStrategy.type, node, zoom, currentLevel, layer);
+    const bestLevel = node.getCoordsForLayer(layer)[0].zoom || node.level;
+    let targetLevel = node.layerUpdateState[layer.id].targetLevel ? node.layerUpdateState[layer.id].targetLevel - 1 :
+        chooseNextLevelToFetch(layer.updateStrategy.type, node, bestLevel, currentLevel, layer);
+
+    targetLevel = Math.min(layer.options.zoom.max, targetLevel);
```

Comment:
This shouldn't be necessary because strategies must use layer.options.zoom.max.

(ie: if you had a case requiring this Math.min, the fix should probably be in the strategy code)

Reply:
moved in strategy code